plugins/repository-s3/build.gradle

```diff
@@ -280,15 +295,26 @@ if (useFixture && minioDistribution) {
 task s3Fixture(type: AntFixture) {
   dependsOn testClasses
   env 'CLASSPATH', "${ -> project.sourceSets.test.runtimeClasspath.asPath }"
+  env 'S3FIXTURE_WORKINGDIR', "${baseDir}"
+  env 'S3FIXTURE_PERMANENT_BUCKET_NAME', "${s3PermanentBucket}"
+  env 'S3FIXTURE_PERMANENT_KEY', "${s3PermanentAccessKey}"
+  env 'S3FIXTURE_TEMPORARY_BUCKET_NAME', "${s3TemporaryBucket}"
+  env 'S3FIXTURE_TEMPORARY_KEY', "${s3TemporaryAccessKey}"
+  env 'S3FIXTURE_TEMPORARY_SESSION_TOKEN', "${s3TemporarySessionToken}"
+  env 'S3FIXTURE_EC2_BUCKET_NAME', "${s3EC2Bucket}"
+  env 'S3FIXTURE_EC2_KEY', "${s3EC2AccessKey}"
+  env 'S3FIXTURE_EC2_TOKEN', "${s3EC2Token}"
```

Comment:
Not so sure about this, compared with your proposal of configuring the fixture using a file. @tlrx what do you think?

Reply:
Reason I changed to env variables - I faced some issues on creating a property file - again due to different stages of configuring / running - property file is not visible to fixture